defaults.yaml

```diff
@@ -3,49 +3,49 @@ Terrain:
 		Type: Sand
 		AcceptsSmudgeType: SandCrater
 		IsWater: False
```

Comment:
Is the TilesetBuilder targeting latest bleed? Then the IsWater entries should be removed.

Reply:
No, I just revived an old branch.